gamezilla/settings.py

```diff
@@ -37,7 +37,7 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
-    "account",
+    "account.apps.AccountConfig",
```

Comment:
Can't we use here only ``account``? Please confirm.

Reply:
Here <app_name>Config is required for signals (In this case for creating profile when user register), not sure if signal configuration with only  <app_name> in INSTALLED_APPS is possible, documentation suggest using this way .